ConcernsCaseWork/ConcernsCaseWork.Service/Configuration/FeatureFlags.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ConcernsCaseWork.Service.Configuration
+{
+	public class FeatureFlags
```

Comment:
Should we maintain all the feature flags in once place or should they life in the project they belong to?

I put the feature flags for the Pages in the website but didn't think too much about other flags for logic. Probably an oversight on my part.

Reply:
Hey Blake, we would need to create a shared project, or we can use the contracts project if that's easier